pandas/io/formats/style.py

```diff
@@ -91,24 +91,24 @@ def _mpl(func: Callable) -> Generator[tuple[Any, Any], None, None]:
 ####
 # Shared Doc Strings
 
-subset = """subset : label, array-like, IndexSlice, optional
+subsets = """subset : label, array-like, IndexSlice, optional
```

Comment:
this one is used in the docstring in 

https://github.com/pandas-dev/pandas/blob/97edd8e4ff89b4a3eedebd03bd718048fef339a3/pandas/io/formats/style.py#L1724

perhaps `subset_arg` would be a more descriptive name?

Reply:
likewise for the other substituted args

Reply:
This has been updated as of: fded48eb7485a986e153a41d28a12bb2ff6493c8